src/v/base/CMakeLists.txt

```diff
@@ -1 +1,7 @@
 v_cc_library(NAME base)
+
+rp_test(
```

Comment:
Should this be in the standard `tests` directory?

Reply:
ah, yeh, thanks. will push an extra commit after we see if this things builds in CI